app/views/incomeTaxComparison/Main.scala.html

```diff
@@ -36,18 +36,26 @@
     </p>
 }
 
+@headingText = @{
+    incomeTaxComparisonViewModel.estimatedIncomeTaxComparisonViewModel.changeInTaxAmount match {
```

Comment:
This pattern is repeated a few times. Perhaps a better abstraction would work instead of always comparing the values. Something like
```
sealed trait TaxComparison {
  def fold[A](gt: => A, lt: => A, eq: => A): A =
    this match {
      case GT => gt
      case LT => lt
      case EQ => eq
}

case object GT extends TaxComparison
case object LT extends TaxComparison
case object EQ extends TaxComparison
```
```
case class EstimatedIncomeTaxComparisonViewModel(items: Seq[EstimatedIncomeTaxComparisonItem]) extends ViewModelHelper {
  ...
  lazy val taxComparison: TaxComparison = {
    changeInTaxAmount match {
      case gt if gt > 0 => GT
      case lt if lt < 0 => LT
      case _ => EQ
    }
  }
  ...
}
```
Then usage would be:
```
incomeTaxComparisonViewModel.estimatedIncomeTaxComparisonViewModel.taxComparison.fold(
  messages("tai.incomeTaxComparison.heading.more"),
  messages("tai.incomeTaxComparison.heading.less"),
  messages("tai.incomeTaxComparison.heading.same")
)
```
This has a few advantages:
- it is more concise
- intent is more explicit
- easier to get right as no longer have to worry about comparisons
- will get exhaustive warnings if you miss a case
- comparison logic is in one place

Let me know if you want to go through anything